.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+
+    tags:
+      - '*'
+
+  pull_request:
+
+env:
+  MAIN_REPO: Oceans-1876/challenger-portal
+
+jobs:
+
+# ----------------------------------------------------------------------
+  # NODE BUILD
+  # ----------------------------------------------------------------------
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - "12"
+          - "14"
+          - "16"
+    steps:
+    # checkout source code
+    - uses: actions/checkout@v2
+
+    # setup node
+    - uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node }}
```

Comment:
We can drop the `matrix` stuff. Considering it's just an app and not a library, we don't need multiple build and can go with just node 14 (the current LTS).

Reply:
I can drop it if you prefer, it's just a nice way to validate against other versions of node to catch things and it wasn't hard to setup (we use it in IN-CORE for our frontend build).

Reply:
The matrix config is useful for publishing libraries, but we don't get much from it for an application that is only run by us in a controlled environment (node v14).